build/build.js

```diff
@@ -2,7 +2,7 @@ const rollup = require('rollup')
 const buble = require('rollup-plugin-buble')
 const commonjs = require('rollup-plugin-commonjs')
 const nodeResolve = require('rollup-plugin-node-resolve')
-const uglify = require('rollup-plugin-uglify')
+const { uglify } = require('rollup-plugin-uglify')
```

Comment:
This change is not related to npm audit

Reply:
Last version of `rollup-plugin-uglify` uses named exports, another option here is to use 
`const uglify = require('rollup-plugin-uglify').uglify`

Reply:
Oh my fault, I though this change was related to whitespaces only. Sorry!